style.css

```diff
@@ -0,0 +1,456 @@
+ * {
```

Comment:
The CSS code has not been indented properly, which makes it hard to read the code.